lib/token.h

```diff
@@ -117,9 +117,13 @@ struct TokenImpl {
     };
     struct CppcheckAttributes *mCppcheckAttributes;
 
+    // For memoization, to speed up parsing of huge arrays #8897
+    enum Cpp11init {UNKNOWN, CPP11INIT, NOINIT} mCpp11init;
```

Comment:
I would like to use `enum class` for new enums

Reply:
Indeed, thanks.
Fixed.